### ISSUE TITLE
Typscript error: Namespace 'NodeJS' has no exported member 'Timeout'.

### ISSUE THREAD

**Issue by gmw-web** [ChrisBrownie55/mini-debounce#4]
When I try to compile a project that uses mini-debounce and do type checking, I get the following error. 

```
ERROR in .../node_modules/mini-debounce/index.d.ts(4,31):
4:31 Namespace 'NodeJS' has no exported member 'Timeout'.
    2 |   callback: Function,
    3 |   timeout: number
  > 4 | ): (...args: any[]) => NodeJS.Timeout | number;
      |                               ^
    5 | export { debounce };
    6 | 
```

**Comment by andria-dev** [comment]
I think I'd need more information about your setup to determine if this is actually the problem, however, it looks like you're using the wrong version of `@types/node`. If you updated it to the latest version, that could potentially fix your problem as stated in the following post:

https://stackoverflow.com/a/56270705/4709300

**Comment by andria-dev** [comment]
Also, if you don't have `@types/node` installed that could be a problem. Although, if you're not using node let me know and I'll see what I can do about making this package independent of `@types/node`

**Comment by gmw-web** [comment]
I'm not using `@types/node` (I just double checked, just to be sure), so that's not the problem.

To provide a bit more context; I get this error when trying to build a Vue application for production.

**Comment by andria-dev** [comment]
Okay, looks like the issue is that to create an isomorphic type for TypeScript to use you can't say `NodeJS.Timeout | number` like I have been. I believe I need to change it to `ReturnType<typeof setTimeout>`.

I'll attempt to fix this now and test it out myself before publishing 👍

**Comment by andria-dev** [comment]
I'm gonna close this as I fixed it in 1.7.0 and there hasn't been any more discourse about it.
